main.cpp

```diff
@@ -52,84 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: car
+//  action 1: car moves Forward
+car.moveForward();
+//  action 2: car ignition turns on
+car.ignitionOn();
+//  action 3: car headlights turn off
+ car.headlightsOff();
 
-//  action 3:
- 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: ship
+//  action 1: ship sounds honk
+ship.honk();
+//  action 2: ship raises flag
+ship.flagRaise();
+//  action 3: ship takes anchor away
+ ship.anchorAway();
```

Comment:
the verb is ` raise`
This should read basically like plain-english with the spaces removed.
`ship.raiseFlag();`
`ship.takeAnchorAway();`
- [ ] revise